docker/images/rolling/Dockerfile

```diff
@@ -98,7 +99,7 @@ RUN sudo apt-get update \
   && sudo apt-get -y dist-upgrade \
   && . /opt/ros/rolling/setup.sh \
   && rosdep update \
-  && rosdep install -i -y --from-path src --skip-keys 'slam_toolbox' \
+  && rosdep install -i -y --from-path src --skip-keys 'turtlebot3_gazebo gazebo_ros_pkgs slam_toolbox' \
```

Comment:
These packages have not yet been released. Also `flatland` and `nav2_map_server` still fail to build.